lib/generators/iiif_print/templates/manifest_builder_service_decorator.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
```

Comment:
TODO - create a separate ticket to revisit the UV manifest. This one is taken from LV hyrax 2.9.6 so it'll prob need to be updated